src/lang/english.txt

```diff
@@ -239,16 +239,16 @@ STR_BUTTON_CATCHMENT                                            :{BLACK}Coverage
 STR_TOOLTIP_CATCHMENT                                           :{BLACK}Toggle coverage area display
 
 STR_TOOLTIP_CLOSE_WINDOW                                        :{BLACK}Close window
-STR_TOOLTIP_WINDOW_TITLE_DRAG_THIS                              :{BLACK}Window title - drag this to move window
-STR_TOOLTIP_SHADE                                               :{BLACK}Shade window - only show the title bar
+STR_TOOLTIP_WINDOW_TITLE_DRAG_THIS                              :{BLACK}Window title{NBSP}-{NBSP}{}Drag this to move window
+STR_TOOLTIP_SHADE                                               :{BLACK}Shade window{NBSP}-{NBSP}Only show the title bar
 STR_TOOLTIP_DEBUG                                               :{BLACK}Show NewGRF debug information
 STR_TOOLTIP_DEFSIZE                                             :{BLACK}Resize window to default size. Ctrl+Click to store current size as default
 STR_TOOLTIP_STICKY                                              :{BLACK}Mark this window as uncloseable by the 'Close All Windows' key. Ctrl+Click to also save state as default
 STR_TOOLTIP_RESIZE                                              :{BLACK}Click and drag to resize this window
 STR_TOOLTIP_TOGGLE_LARGE_SMALL_WINDOW                           :{BLACK}Toggle large/small window size
-STR_TOOLTIP_VSCROLL_BAR_SCROLLS_LIST                            :{BLACK}Scroll bar - scrolls list up/down
-STR_TOOLTIP_HSCROLL_BAR_SCROLLS_LIST                            :{BLACK}Scroll bar - scrolls list left/right
-STR_TOOLTIP_DEMOLISH_BUILDINGS_ETC                              :{BLACK}Demolish buildings etc. on a square of land. Ctrl selects the area diagonally. Shift toggles building/showing cost estimate
+STR_TOOLTIP_VSCROLL_BAR_SCROLLS_LIST                            :{BLACK}Scroll bar{NBSP}-{NBSP}Scrolls list up/down
+STR_TOOLTIP_HSCROLL_BAR_SCROLLS_LIST                            :{BLACK}Scroll bar{NBSP}-{NBSP}Scrolls list left/right
+STR_TOOLTIP_DEMOLISH_BUILDINGS_ETC                              :{BLACK}Demolish{NBSP}-{NBSP}{}Clear all objects on a selected area. Ctrl+Click+Drag selects the area diagonally. Shift+Click shows estimated cost without purchase
```

Comment:
This was made on purpose as a standard for most tooltips. It ensures you do not get any weird fragmentation after the hyphen (which happens frequently if the TT name is too long) and it makes the tooltip name nice and centered. I personally find the final result to be pleasing aesthetically.

![1](https://user-images.githubusercontent.com/3621746/71493555-88b38480-2837-11ea-9e77-6acb81b2e8e0.jpg)
![2](https://user-images.githubusercontent.com/3621746/71493557-894c1b00-2837-11ea-91f6-e6bdb3a582c6.jpg)
![3](https://user-images.githubusercontent.com/3621746/71493558-894c1b00-2837-11ea-8163-096eb7a8b952.jpg)
![4](https://user-images.githubusercontent.com/3621746/71493559-894c1b00-2837-11ea-9b41-f2d8b88b833c.jpg)

Reply:
Ok I get the idea, but in this case a newline without the hyphen should work too

Reply:
The aim of the hyphen is to provide an aesthetically pleasing divider. Colons were another option, and although perhaps better suited for newline, they do not look as nice IMO.

Reply:
Going back to this, aside from preventing bad fragmentation, another issue here is that because all the tooltips text is in bold (tool name + description), there needs to be some form of separation telling tool name from description, and the line break alone does not do it very well if, for example, the tool name has the same length or is wider than the description line(s). Assuming some of you find the hyphen a bit jarring, would you find it more agreeable if it were to be replaced with colons (":")? This would also simplify editing since it would only require 1 linebreak and no non-breaking spaces, greatly reducing the number of tags required.